flags-sdk/launchdarkly/README.md

```diff
@@ -0,0 +1,94 @@
+
+If you deployed your own and configured the feature flags on LaunchDarkly, you can also use the [Flags Explorer](https://vercel.com/docs/workflow-collaboration/feature-flags/using-vercel-toolbar) to test different variations by creating overrides.
+
+## Deploy this template
+
+The easiest way to get started with LaunchDarkly is through the native integration in the [Vercel Marketplace](https://vercel.com/marketplace/launchdarkly).
+
+[![Deploy with Vercel](https://vercel.com/button)]()
+
+### Step 1: Link the project
+
+In order to use the Flags Explorer, you need to link the project on your local machine.
+
+```bash
+vercel link
+```
+
+Select the project from the list you just deployed.
+
+### Step 2: Pull all environment variables
+
+This allows the Flags SDK and the Flags Explorer to work correctly, by getting additional metadata.
+
+```bash
+vercel env pull
+```
+
+### Step 3: Create Feature Flags
+
+Head over to the [LaunchDarkly Console](https://app.launchdarkly.com) and create the feature flags and experiments required by this template.
+
+Be sure to enable the `SDKs using Client-side ID` option for each Feature Flag.
+
+Feature Flags:
+
+- `Summer Sale` (type boolean) with the key `summer-sale` and the variations `true` and `false`.  Edit the default targeting rule to serve a percentage rollout with a 50/50 split by `user.key`.
+- `Free Delivery` (type boolean) with the key `free-delivery` and the variations `true` and `false`.  Edit the default targeting rule to serve a percentage rollout with a 50/50 split by `user.key`.
+- `Proceed to Checkout` (type string) with the key `proceed-to-checkout` and the following variations:
+  - Name: `Control`, Value: `blue`
+  - Name: `Test`, Value: `green`
+  - Name: `Test #2`, Value: `red`
+
+You can also find the gate ids in the `flags.ts` file.
+
+Ensure all Flags are ON.
+
+### Step 4: Configure the Experiment
+
+Create the `Proceed to Checkout` experiment:
+
+- Name: `Proceed to Checkout`
+- Hypothesis: `Button color influences rate at which customers proceed to checkout`
+- Type: `Feature change`
+- Randomization Unit: `user`
+- Randomization Attribute: `key`
+- Metric: Create a new metric:
+  - Event kind: `Custom`
+  - Event key: `proceed-to-checkout-clicked`
+  - What do you want to measure? `Occurence (conversion: binary)`
+  - Metric definition: `Percentage of user units that sent the event, where higher is better`
+  - Metric name: `Proceed to Checkout Clicked`
+- Varaiations: Choose flag `proceed-to-checkout`
+- Audience: 
+  - In this experiment: Choose `Custom` and enter `100`%
+  - Split audience: Choose `Split equally`
+- Statistical approach: Default values
+
+After that, start the Experiment.
+
+### Step 6 (optional): Set additional environment variables
+
+If you provide the `LAUNCHDARKLY_API_KEY`, `LAUNCHDARKLY_PROJECT_KEY` and `LAUNCHDARKLY_ENVIRONMENT` environment variables, the Flags Explorer will fetch additional metadata from the LaunchDarkly API.
```

Comment:
We should provide a `.env.example` in the project with placeholders for what's needed and update the list here

In testing I found that using the adapter and flags explorer required FLAGS_SECRET and EDGE_CONFIG

And client usage required a NEXT_PUBLIC_LAUNCHDARKLY_CLIENT_SIDE_ID